components/widgets/forest-change/tree-cover-gain-simple/index.js

```diff
@@ -24,7 +24,7 @@ export default {
   metaKey: 'umd_tree_cover_gain_from_height',
   dataType: 'gain',
   pendingKeys: ['threshold'],
-  refetchKeys: ['threshold'],
+  refetchKeys: ['threshold', 'baselineYear'],
```

Comment:
I think I never understood this, what do we use `refetchKeys` for?

Reply:
sometimes we don't need to refetch because the first request get all info we need, so the only thing needed is filter the data displayed. However, for some widgets, every time the user changes the settings, we need to refetch passing a new value to the query. It should be always, but for some reason it's not.